### PR TITLE
Docs: Fix links

### PR DESCRIPTION
**Pull request by jonsuh**
Fix links in docs: remove `localhost:8000`

### CODE REVIEW

File: docs/docs/production-app.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ PageRenderer's constructor [loads the page resources](/docs/production-app/#load
...
+This occurs in [loader.js](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js). The main function here is [getResourcesForPathname()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L268). Given a path, it will find its page, and import its component module json query results. But to do this, it needs access to that information. This is provided by [async-requires.js](http://localhost:8000/docs/write-pages/#async-requiresjs) which contains the list of all pages in the site, and all their dataPathss. [fetchPageResourcesMap()](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L33) takes care of requesting that file, which occurs the [first time](https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby/cache-dir/loader.js#L292) `getResourcesForPathname()` is called.
```
Comment (fk) [inline docs/docs/production-app.md]: Hey @jonsuh, thanks for helping out! 👋 

This paragraph contains one more `localhost:8000`, could you please fix that one, too? 🙏

Comment (jonsuh) [reply docs/docs/production-app.md]: @fk Sure, thanks for catching that! Apparently overlooked it. Fixed in https://github.com/gatsbyjs/gatsby/pull/8276/commits/10c979e4360ebbf08ae10fffd3436d062da9f75e

Comment (fk) [reply docs/docs/production-app.md]: Thank you Jonathan! 🙏

**Comment by Moocar** [comment]
Thanks @jonsuh !

**Pull request merged**